IBM/openapi-validator
# feat: synchronize the inCodeValidator output to match the CLI -json

Purpose:
- Both the inCodeValidator and CLI -json return objects, so we want to standardize and keep these formats in sync.
- Make output changes to fewer places when introducing new features. Simplify the code base.

Changes:
- Add `infos` and `hints` to the JSON output.
- Split the object creation and the printing of the JSON object into separate functions, so the `inCodeValidator` may utilize the same object-creation function.

Tests:
- Add a test utility that aggregates the inCodeValidator errors, warnings, etc. into a single array to convert output to a format that is still compatible with existing tests.
- Remove checks that expect the `infos` and `hints` fields to be undefined.
